### PR TITLE
admin-guide:image-policy: allowed registries for import

### LINKED ISSUE #7123: AllowedRegistriesForImport format is undocumented
It's mentioned here:
https://docs.openshift.org/latest/install_config/master_node_configuration.html#master-config-image-policy-config

But no example or discussion is provided to help people understand how to actually craft a proper definition of allowed registries.  They have to go to the api to understand it:
https://github.com/openshift/origin/blob/00c65897e60f396b716b68011d691f5c0675a4c0/pkg/cmd/server/api/v1/types.go#L425-L437

@miminar since you're working on this logic anyway, can you add some appropriate doc?
It should probably be added either in this section:
https://docs.openshift.org/latest/admin_guide/image_policy.html

or as a peer to that same section.

/cc @bmcelvee

### PR DESCRIPTION
**Pull request by miminar**
Documents feature/bugfix implemented/merged in openshift/origin#17783

Resolves #7123

### CODE REVIEW

**Comment by miminar** [comment]
@bmcelvee can you please take a look?

**Comment by bmcelvee** [comment]
Thanks, @miminar! I'll go ahead and get this merged today and open a follow-up PR with some minor edits. Looks like this should go into 3.9. Is that correct?

**Comment by miminar** [comment]
That's correct, but wait. I just realized I need to mention one or two things there.

**Comment by bmcelvee** [comment]
Sure thing! 🌞

**Comment by miminar** [comment]
Done. Unless @bparees wants to take another look, it's all in your hands :-)

File: admin_guide/image_policy.adoc
Comment (bparees) [inline admin_guide/image_policy.adoc]: registries typo

File: admin_guide/image_policy.adoc
Comment (bparees) [inline admin_guide/image_policy.adoc]: that's not true is it?  by default you get the registries we hardcoded as defaults for the whitelist.

**Review by bparees — CHANGES_REQUESTED** [review]

File: admin_guide/image_policy.adoc
Comment (miminar) [reply admin_guide/image_policy.adoc]: You're right. It's the ansible-installer that doesn't write this policy into `master-config.yaml`. I totally forgot about `openshift start master --write-config` which generates the setting.

I'll open another PR against openshift-ansible to write the policy by default.

Comment (miminar) [reply admin_guide/image_policy.adoc]: I just removed the *default* part.

**Comment by bparees** [comment]
lgtm

**Comment by miminar** [comment]
Ansible PR: openshift/openshift-ansible#7788

Once merged, I'll do a doc follow-up with amendments for advanced installation..

**Comment by miminar** [comment]
@bmcelvee it's all yours now :bowing_man:

**Comment by bmcelvee** [comment]
Thanks, @miminar! :sunflower: Do I need to wait for the associated PRs to merge, or is it all set?

**Pull request merged**
